### PR TITLE
Create outline for Montreal City file

### LINKED ISSUE #110: MGineer issue
# Testing
- [x] Create a branch
- [x] add commits on my own branch
- [x] open a pull request
- [x] collaborate and have a conversation
- [x] make some more commits

### PR DESCRIPTION
**Pull request by MGineer**
This pull request adds a folder and file for Montreal City. 

Do you have a favorite type of food that I should add?

This pull request resolves #110 :hand:

### CODE REVIEW

File: MGineer/MGineer-Montreal.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Montreal City
+
+### Restaurants
+
```
Comment (smanthena) [inline MGineer/MGineer-Montreal.md:8]: Any pizza place suggestions?

**Comment by mccue** [comment]
Montreal is a really nice city! D'accord?

File: MGineer/MGineer-Montreal.md (hunk 1 above)
Comment (dany-phantom) [inline MGineer/MGineer-Montreal.md:8]: @MGineer Places to watch :it: vs :es:?

Comment (MGineer) [inline MGineer/MGineer-Montreal.md:8]: @smanthena  

### Here's a list of places for great pizza:
- http://www.pizzanino.com/
- http://www.restaurantszibo.com/en
- http://bevopizza.com/en/

**Comment by kmmtechnology** [comment]
which team do u support Italy /Spain?

**Comment by MGineer** [comment]
oui oui

File: MGineer/MGineer-Montreal.md (hunk 1 above)
Comment (matteocellucci) [inline MGineer/MGineer-Montreal.md:8]: @smanthena Try Naples 🇮🇹

Comment (MGineer) [inline MGineer/MGineer-Montreal.md:8]: @dmahmalat http://lespyrenees.ca/

**Comment by dany-phantom** [comment]
Azzurri !

**Pull request merged**